.github/matic-cli-config.yml

```diff
@@ -20,6 +21,7 @@ ethURL: http://ganache:9545
 ethHostUser: ubuntu
 devnetType: docker
 borDockerBuildContext: "../../bor"
-heimdallDockerBuildContext: "https://github.com/maticnetwork/heimdall.git#develop"
+heimdallDockerBuildContext: "https://github.com/shibaone/heimdall.git#develop"
 sprintSizeBlockNumber:
   - '0'
+# devnetBorFlags: config,config,config
```

Comment:
Why this is commented ?

Reply:
This line was not there in the config but I added it based on how matic cli config is being used by https://github.com/maticnetwork/bor. So I later commented it out.